packages/web-components/@sfx/products/test/index.test.ts

```diff
@@ -0,0 +1,97 @@
+import { TemplateResult } from 'lit-element';
+import { expect, sinon, spy, stub } from './utils';
+import Products from '../src/products';
+
+describe('Products Component ', () => {
+  let component;
+  beforeEach(() => {
+    component = new Products();
+  });
+
+  describe('general', () => {
+    it('should default to have an empty array of products', () => {
+      expect(component.products).to.deep.equal([]);
+    });
+
+    it('should default to 12 maxItems', () => {
+      expect(component.maxItems).to.equal(12);
+    });
+  });
+
+  describe('connectedCallback', () => {
+    it('should call super', () => {
+      const superConnected = stub(component.__proto__, 'connectedCallback');
```

Comment:
I would prefer to use `Object.getPrototypeOf(component)` instead of `__proto__` since the latter is just there for compatibility. The other tests go to the parent class directly and stub its prototype, but I think this is a more resilient approach.